avro/src/main/java/com/fasterxml/jackson/dataformat/avro/AvroTypeDeserializer.java

```diff
@@ -57,6 +54,13 @@ public Object deserializeTypedFromScalar(JsonParser p, DeserializationContext ct
 
     @Override
     public Object deserializeTypedFromAny(JsonParser p, DeserializationContext ctxt) throws IOException {
+        AnnotationIntrospector ai = ctxt.getAnnotationIntrospector();
```

Comment:
The whole idea of type (de)serializers is that they can be composed with content (de)serializers and work independently. But use of type (de)serializers is inferred from polymorphic handling, so perhaps that should not be enabled if no type handling is desired?